src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -47,12 +52,89 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
     reset({ reviewNotes: '' });
   };
 
+  const overflow: object = {
+    'overflow-y': 'scroll',
+    'max-height': '300px'
+  };
+
+  const proposedSolutionStyle = {
+    cursor: 'pointer',
+    width: 'auto',
+    margin: 'auto',
+    display: 'block'
+  };
+
+  if (cr.type === 'ISSUE') {
```

Comment:
if you look at the ChangeRequestType definition you'll see that there are 5 types:   

Issue: 'ISSUE',
Redefinition: 'DEFINITION_CHANGE',
Other: 'OTHER',
StageGate: 'STAGE_GATE',
Activation: 'ACTIVATION'

An Issue, Redefinition, and Other can all have proposed solutions. It's just stage gate and activations that don't

Reply:
incorporated